ctms/dependencies.py

```diff
@@ -19,6 +19,7 @@ def get_settings() -> Settings:
 
 
 def get_db():  # pragma: no cover
+    SessionLocal = session_factory()
```

Comment:
```python
def session_factory(scoped: bool = False):
    factory = sessionmaker(autocommit=False, autoflush=False, bind=get_engine())
```

Does this mean we're going to create a new engine every time a route fetches a DB connection with this dependency? Something from the SQLAlchemy [docs](https://docs.sqlalchemy.org/en/14/core/connections.html):

> The typical usage of [create_engine()](https://docs.sqlalchemy.org/en/14/core/engines.html#sqlalchemy.create_engine) is once per particular database URL, held globally for the lifetime of a single application process. A single [Engine](https://docs.sqlalchemy.org/en/14/core/connections.html#sqlalchemy.engine.Engine) manages many individual [DBAPI](https://docs.sqlalchemy.org/en/14/glossary.html#term-DBAPI) connections on behalf of the process and is intended to be called upon in a concurrent fashion.